build.sbt

```diff
@@ -69,7 +69,8 @@ def bintraySettings: Seq[Setting[_]] = Seq(
   bintrayOrganization := Some("lagom"),
   bintrayRepository := "sbt-plugin-releases",
   bintrayPackage := "lagom-sbt-plugin",
-  bintrayReleaseOnPublish := false
+  bintrayReleaseOnPublish := false,
+  credentials := List(Path.userHome / ".bintray").filter(_.exists).map(Credentials(_))
```

Comment:
IIUC this is checking the path `~/.bintray` exists, but not that there's a `.credentials` file in it or that it contains valid credentials. What am I missing ?